package.json

```diff
@@ -609,6 +609,11 @@
           "default": true,
           "description": "Loads user and system-wide PowerShell profiles (profile.ps1 and Microsoft.VSCode_profile.ps1) into the PowerShell session. This affects IntelliSense and interactive script execution, but it does not affect the debugger."
         },
+        "powershell.enableReferencesCodeLens": {
+          "type": "boolean",
+          "default": false,
```

Comment:
I would err on defaulting to `true` as that's the current experience, and this new setting lets users disable it when they want to improve performance for huge workspaces.

Reply:
Yeah I was hoping to default this to `false` until we fix the code lens. The majority of folks impacted by the perf won't know to look for this setting :/

I'll go ahead and make that change (default to `true`) and just prioritize fixing the code lens then.